app/code/Magento/InventoryCatalog/etc/module.xml

```diff
@@ -8,7 +8,7 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Module/etc/module.xsd">
     <module name="Magento_InventoryCatalog" setup_version="2.0.0">
         <sequence>
-            <module name="InventorySales"/>
+            <module name="Magento_InventoryAPI"/>
```

Comment:
Looks like is mistake
In this config we need to describe sequence but not dependency